runtime/onert/core/src/compiler/ShapeValidator.cc

```diff
@@ -1144,5 +1144,22 @@ void ShapeValidator::visit(const ir::operation::RmsNorm &node)
               (gamma_shape.dim(0) == ifm_shape.dim(ifm_shape.rank() - 1)));
 }
 
+void ShapeValidator::visit(const ir::operation::RoPE &node)
+{
+  const auto &operands = _graph.operands();
+  const auto ofm_index{node.getOutputs().at(0)};
+  if (operands.at(ofm_index).info().isDynamic())
+    return;
```

Comment:
This lines seems to say "For dynamic shape output, I will not check anything.", and other operations (not only this PR) do same pattern. However, personally I prefer to check the possible conditions which is true even for dynamic shape output.

@ys44kim What happens if we remove these lines? I guess it may be okay even for dynamic output case.

Reply:
As you know, it is the same pattern code as other validators.
I couldn't check if it was valid for the case of dynamic shape , so I kept it the same as others.
If it's okay to check more and delete it as your opinion, I'll upload it to another PR.
Thank you.